pallets/automation-time/src/tests.rs

```diff
@@ -2252,7 +2276,7 @@ fn auto_compound_delegated_stake_does_not_reschedule_on_failure() {
 	new_test_ext(START_BLOCK_TIME).execute_with(|| {
```

Comment:
Doesn’t this test need update too? `auto_compound_delegated_stake_does_not_reschedule_on_failure`

Reply:
We need a few more tests, for example, 
the next execution will not be scheduled when the wallet has left all delegations.

Reply:
I have add test cases for 4 scenarios

Reply:
Great, the test cases look clearer now.